src/Presentation/Factory/EventFactory.php

```diff
@@ -134,6 +135,21 @@ private function getLocationProperties(Event $event): Generator
 
         if ($event->getLocation()->hasGeographicalPosition()) {
             yield new Property('GEO', new GeoValue($event->getLocation()->getGeographicPosition()));
+            $appleLocationProperty = new Property(
+                'X-APPLE-STRUCTURED-LOCATION',
+                new TextValue(sprintf('geo:%1.6F,%1.6F',
+                    number_format($event->getLocation()->getGeographicPosition()->getLatitude(), 6),
+                    number_format($event->getLocation()->getGeographicPosition()->getLongitude(), 6),
+                )),
+                [
+                    new Parameter('VALUE', new TextValue('URI')),
+                    new Parameter('X-ADDRESS', new TextValue((string) $event->getLocation())),
+                    new Parameter('X-APPLE-RADIUS', new IntegerValue(49)),
+                    new Parameter('X-TITLE', new TextValue('')),
```

Comment:
Can you add a title property to `\Eluceo\iCal\Domain\ValueObject\Location`?

Reply:
Done